2-Authorization-I/1-call-graph/SPA/src/fetch.js

```diff
@@ -1,3 +1,8 @@
+import { BrowserAuthError } from '@azure/msal-browser';
+import { msalInstance } from './index';
+import { protectedResources, msalConfig } from '../src/authConfig';
+import { addClaimsToStorage } from './utils/storageUtils';
+
 /*
  * Copyright (c) Microsoft Corporation. All rights reserved.
```

Comment:
```suggestion
 * Copyright (c) Microsoft Corporation. All rights reserved.
```

This should appear at the very top